indico/modules/designer/pdf.py

```diff
@@ -92,22 +94,33 @@ def get_pdf(self):
         return data
 
     def _draw_item(self, canvas, item, tpl_data, content, margin_x, margin_y):
-        style = ParagraphStyle({})
-        style.alignment = ALIGNMENTS[item['text_align']]
-        style.textColor = item.get('color') or '#000000'
-        style.backColor = item.get('background_color') or None
-        style.borderPadding = (0, 0, 4, 0)
-        style.fontSize = _extract_font_size(item['font_size'])
-        style.leading = style.fontSize
+        styles = {
+            'alignment': ALIGNMENTS[item['text_align']],
+            'textColor': item.get('color') or '#000000',
+            'backColor': item.get('background_color') or None,
+            'borderPadding': (0, 0, 4, 0),
+            'fontSize': _extract_font_size(item['font_size'])
+        }
+        styles['leading'] = styles['fontSize']
```

Comment:
I'd just call `_extract_font_size` twice or assign `font_size = _extract_font_size(...)` and then use that in the dict literal